### PR TITLE
PO-41: Move thermostat ui

### PR DESCRIPTION
**Pull request by itolegmysin**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -124,7 +124,9 @@ module.exports = ({stage, target, server, coverage} = {}) => ({
         }),
         new CopyWebpackPlugin([
             { from: 'src/images/favicon.ico', to: 'favicon.ico' },
-            { from: 'src/locales', to: 'locales' }
+            { from: 'src/locales', to: 'locales' },
+            { from: 'src/images', to: 'static/img' }
```
Comment (khenderick) [inline webpack.config.js:128]: Please make sure that these changes are tested on both cloud as gateway targets.

**Comment by iOlivier** [comment]
@khenderick @svanscho 
Is there an url directly to a gateway that we can use to test the gateway API? Because Oleg can't access his gateway since he's working from home.

**Comment by svanscho** [comment]
Oleg should have gotten Anastasia's gateway.
He can use this for testing.

Gr. S

**Comment by khenderick** [comment]
@iOlivier No, that's not possible, he'll need to bring the Gateway home or check at the office whether VPN access or port forwarding is possible.

**Comment by iOlivier** [comment]
Let's merge this and lookinto it once we have fysical

**Comment by iOlivier** [comment]
FYI. Oleg picked up his gateway so he will be able to test it soon.

File: webpack.config.js (hunk 1 above)
Comment (itolegmysin) [reply webpack.config.js:128]: Unfortunately I don't have thermostats (itolegmysin@gmail.com).
image
Btw, webpack building without any issues.

Comment (svanscho) [reply webpack.config.js:128]: @itolegmysin you will have to configure a thermostat first

**Pull request merged**